### DISCUSSION TITLE
Seeking Information on Swallowing Screening for Dysphagia (Difficulty swallowing) in MIMIC-IV

### DISCUSSION THREAD

**Discussion by Chenghongtao99** [MIT-LCP/mimic-code#1538] [MIMIC-IV] [answered]
Dear MIMIC-IV Community Members,

I am currently conducting a study related to dysphagia (Difficulty swallowing), utilizing data from the MIMIC-IV database. In my research, dysphagia was extracted from the "chartevents" table using itemid=225118. However, one of the reviewers of my work has inquired about the specific test or criteria used to diagnose dysphagia (methods).

I have searched the MIMIC-IV documentation and related publications but have been unable to find detailed information regarding the assessment method used for bedside swallowing screening. Examples of dysphagia assessments include methods such as videofluoroscopic swallow study (VFSS), fiberoptic endoscopic evaluation of swallowing (FEES), clinical bedside swallowing assessment, and water swallow test. I was hoping someone in this community might have insights into the specific test or criteria used to diagnose dysphagia in the MIMIC-IV database and any information regarding the timing of the assessment.

Any additional information about the reproducibility and properties of the assessment would also be highly appreciated. Your insights will be invaluable for addressing the reviewer's concerns and ensuring the accuracy of my research.

Thank you in advance for your assistance and expertise.

**Comment by Chenghongtao99** [comment]
![image](https://user-images.githubusercontent.com/132259408/236425264-8bb93822-4c29-41d7-ac0d-2f84572f8a46.png)

**Reply by alistairewj** [reply, marked as answer]
Here is the process from the nursing team:

- Complete a swallowing screening 
- Complete an O-cat score
- Assess their alertness, ability to remain upright and oropharyngeal secretions
- Is their breathing stable?
- Does their PMH include:
    - history of dysphagia or a diet of thickened liquids
- If yes to any of the above questions: Obtain NPO order. No food or drink until a consult.
- If no to all questions: proceed to a water swallow screening.

**Reply by Chenghongtao99** [reply]
Thank you so much for your swift and comprehensive response. It is indeed helpful to understand the nursing team's process in assessing dysphagia.
